addons/mass_mailing/models/res_config_settings.py

```diff
@@ -11,6 +11,9 @@ class ResConfigSettings(models.TransientModel):
     mass_mailing_outgoing_mail_server = fields.Boolean(string="Specific Mail Server", config_parameter='mass_mailing.outgoing_mail_server',
         help='Use a specific mail server in priority. Otherwise Odoo relies on the first outgoing mail server available (based on their sequencing) as it does for normal mails.')
     mass_mailing_mail_server_id = fields.Many2one('ir.mail_server', string='Mail Server', config_parameter='mass_mailing.mail_server_id')
+    show_blacklist_buttons = fields.Boolean(string="Show blacklist buttons on unsubscribe page",
```

Comment:
Why is it a parameter ?

Reply:
what could it be ? implied group ? Well, it thought it was easier to use a parameter.